src/Adapters/Logger/WinstonLoggerAdapter.js

```diff
@@ -0,0 +1,106 @@
+import { LoggerAdapter } from './LoggerAdapter';
```

Comment:
this whole file is most of the contents of FileLoggerAdapter (above).  I remove the file location stuff that wasn't doing anything.

I think it makes sense to move the file location stuff and other file logger configuration stuff that currently lives in src/logger.js into FileLoggerAdapter.js, but I didn't bite that off with this pr.